src/main/java/io/github/jbellis/brokk/prompts/CodePrompts.java

```diff
@@ -74,6 +81,7 @@ public final List<ChatMessage> collectCodeMessages(IContextManager cm,
         messages.addAll(cm.getWorkspaceReadOnlyMessages());
         messages.addAll(originalWorkspaceEditableMessages);
         messages.addAll(parser.exampleMessages());
+        messages.add(elidedBlocksMessage());
```

Comment:
After the S/R examples (so elision guide follows naturally with the context above), and immediately before history and task messages where elided blocks appear (so it has fresh memory about it and is hopefully less likely to assume that elision blocks are the S/R format).